timetable.md

```diff
@@ -29,3 +29,28 @@
 
 - **75** [Exercise: SSE Git cheat sheet](https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/version-control/material/cheat_sheet_text.md)
 - **15** Discussion
+
+## 3.1 -- Thu, November 04, 2021
+
+- **10** Orga
+- **10** Introduction to virtualization
+- **30** Virtualbox with practical example
```

Comment:
seems like this is not yet updated. there are multiple slides for this entry, right?

Reply:
It is not properly updated yet. I am not sure what you mean with "there are multiple slides for this entry"? Which entry? The Virtualbox one?

Reply:
I saw the update. Problem solved.